website/src/components/props-table.tsx

```diff
@@ -141,39 +63,45 @@ Remove the use of <PropsTable of="${of}" /> for this component in the docs.`,
           <MDXComponents.th>Type</MDXComponents.th>
           <MDXComponents.th>Description</MDXComponents.th>
           <MDXComponents.th>Default</MDXComponents.th>
-          {/* <MDXComponents.th>Required</MDXComponents.th> */}
         </tr>
       </thead>
       <tbody>
-        {entries.map(([propName, values]) => (
-          <tr key={propName}>
-            <MDXComponents.td>{propName}</MDXComponents.td>
+        {propList.map((prop) => (
+          <tr key={prop.name}>
+            <MDXComponents.td>{prop.name}</MDXComponents.td>
             <MDXComponents.td>
+              {prop.required && (
+                <Tag
+                  size="sm"
+                  colorScheme="red"
+                  mr={1}
+                  verticalAlign="baseline"
+                >
+                  required
+                </Tag>
```

Comment:
I really like that we visualise the required props.

From a design perspective this looks a bit too heavy, but I have no other proposal at hand.
Maybe we can iterate on that =)

Reply:
Does shrinking it down a little like this help?
Seems like required props are quite rare in chakra, so the tag won't be seen often or in big concentration anywhere.

<img width="772" alt="Screenshot 2021-02-04 at 20 41 13" src="https://user-images.githubusercontent.com/14360171/106946660-adac4380-6729-11eb-8e9d-b4a57e0548b7.png">